apps/web/src/actions/evaluations/runBatch.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  DatasetsRepository,
+  DocumentVersionsRepository,
+  EvaluationsRepository,
+} from '@latitude-data/core/repositories'
+import { queues } from '@latitude-data/jobs'
+import { nanoid } from 'nanoid'
+import { z } from 'zod'
+
+import { authProcedure } from '../procedures'
+
+export const runBatchAction = authProcedure
+  .createServerAction()
+  .input(
+    z.object({
+      datasetId: z.number(),
+      projectId: z.number(),
+      documentUuid: z.string(),
+      commitUuid: z.string(),
+      runCount: z.number(),
+      offset: z.number().optional().default(0),
```

Comment:
Also limit no?

Reply:
And also a boolean to pick all the rows
<img width="794" alt="image" src="https://github.com/user-attachments/assets/ce5386ef-4738-485b-9c3b-cdbcc2668510">

Reply:
yes

Reply:
it's gonna be fromLine and toLine btw

Reply:
> And also a boolean to pick all the rows

better to handle this in frontend, don't send the limits to the backend if the users wants everything